tests/model_enums/tests.py

```diff
@@ -155,6 +156,23 @@ def test_templates(self):
         output = template.render(Context({'Suit': Suit}))
         self.assertEqual(output, 'Diamond|1')
 
+    def test_special_attributes_and_properties_can_be_members(self):
```

Comment:
As there are two blocks with comments in this function, wouldn't it be better to make two test functions and remove comments ?

Reply:
Maybe, but in essence they're tests to fix exactly the same issue (although the fix is slightly different for each).

I'll leave it to one of the fellows to decide when they review.